File: Basic/Check if A Number is Prime or Not/SolutionByKaran.py
def prime(a):
	i=1
	k=0
	
	while(i<=a):
		
		if(a%i==0):
			
      k=k+1
		
    i=i+1
		
	if(k==2):
		
    print("Yes")
			
	else:
		
    print("No")
		
print("Enter the number: ")
n = int(input())
prime(n)

File: Basic/Check if A Number is Prime or Not/SolutionByKaran.cpp
#include <iostream>
using namespace std;
void prime(int n)
{
    int k=0;
    for(int i=1;i<=n;i++)
    {
          if(n%i==0)
          {
              k++;
          }
    }
    if(k==2)
    {
          cout<<"Yes";
    }
    else
    {
          cout<<"No";
    }
	
}
int main() 
{	
    int a;
    cout<<"Enter the number: \n";
    cin>>a;
    prime(a);
    return 0;
}
